open-metadata-implementation/user-interfaces/access-services-user-interface/pom.xml

```diff
@@ -145,18 +145,15 @@
         <dependency>
             <groupId>org.odpi.egeria</groupId>
             <artifactId>connector-configuration-factory</artifactId>
-            <version>1.1-SNAPSHOT</version>
             <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>org.odpi.egeria</groupId>
             <artifactId>repository-services-apis</artifactId>
-            <version>1.1-SNAPSHOT</version>
-        </dependency>
+                  </dependency>
```

Comment:
minor - indentation?

Reply:
There seemed to be a tab error in the original - this change corrects the indentation (even though the diff looks strange)